mantidproject/mantid
# Remove API::MemoryManager

MemoryManager is a historic artefact and does nothing in the current implementation, so this PR removes it.

**To test:**

Code review & automatic tests should be sufficient.

Fixes #16487.

_Internal change, does not need to be in the release notes._
(There is a change of one menu item in MantidPlot, however it was compiled only when a certain TCMALLOC define was set, and I think it was thus not visible in any recent Manti release anyways).

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
